docs/YouboraSessions.md

```diff
@@ -0,0 +1,16 @@
+# Youbora sessions integration
+
+## To integrate youbora sessions follow this documentation:
+https://documentation.npaw.com/npaw-integration/docs/application-analytics-custom-events-data-collection
```

Comment:
need to add part on how to pass plugin instance to player on the plugin config

Reply:
I need plugin config creation sample to add that part

Reply:
https://github.com/kaltura/kaltura-player-android-samples/blob/develop/AdvancedSamples/Youbora/app/src/main/java/com/kaltura/playkit/samples/youbora/MainActivity.kt#L287